src/einsteinpy/examples.py

```diff
@@ -0,0 +1,26 @@
+"""Example data.
+"""
+import astropy.units as u
+import numpy as np
+
+from einsteinpy.bodies import Body
+from einsteinpy.coordinates import SphericalDifferential
+from einsteinpy.geodesic import Geodesic
+
+
+def _perihelion():
+    Attractor = Body(name="BH", mass=6e24 * u.kg, parent=None)
+    sph_obj = SphericalDifferential(
+        130 * u.m,
+        np.pi / 2 * u.rad,
+        -np.pi / 8 * u.rad,
+        0 * u.m / u.s,
+        0 * u.rad / u.s,
+        1900 * u.rad / u.s,
+    )
+    Object = Body(differential=sph_obj, parent=Attractor)
+    geodesic = Geodesic(body=Object, time=0 * u.s, end_lambda=0.002, step_size=5e-8)
+    return geodesic
+
+
+perihelion = _perihelion()
```

Comment:
Please don’t do this. Please don’t call the function yourself and let the user call the function. As you add more examples in this, importing any one of them would cause all of them to run unnecessarily. And the runtimes are not so small to be ignored.

I discussed this point when you were adding predefined bodies to bodies module

Reply:
Oh. For this, I did not worry because, this is just example snippet. It will *never* run from within the codebase! Why care about it's performance?

Reply:
I mean, its just for demo purpose :p

Reply:
> Oh. For this, I did not worry because, this is just example snippet. It will _never_ run from within the codebase! Why care about it's performance?

My point being, as we add more examples, running one of them runs all of them. If you don’t believe me, write a small file, add three functions in it the same way you did here, and then try to import one of them. 


Personally, I am in very strong opposition to this. And I say, we fix this up in bodies module also.

Anyway, APIs are supposed to provide classes and functions, not objects.

Reply:
I totally understand what you are saying, but we will think about it when we will add another example (which we are not doing in next coming year atleast)

Reply:
One more example, this is gonna change. Don't whine about API breaks then 😆 .

I can't imagine someone waiting for geodesic calculation, when he actually wanted to run a symbolic example.

Reply:
Furthermore, let me point out another problem with this. Codecov marks those lines green, when the code is actually not covered.

It's inflated coverage.